test/benchmark.hs

```diff
@@ -1,7 +1,15 @@
 {-
-  Install microbench to build this program:
+  This program should generally be run using `cabal bench` or
+  `stack bench`. To use `stack bench`, edit stack.yaml to include
+
+  extra-deps:
+  - microbench-0.1
+
```

Comment:
I would be OK with that being added to the `stack.yaml` file (or would this result in anyone installing FGL as a dependency via stack requiring microbench? I admittedly tend to prefer cabal-install for development.)